### PR TITLE
Fixed titles in views.

### PR DESCRIPTION
**Pull request by ghost**
Get titles in views from model.Meta.
For listing-view as plural.
For edit-view as singular.

### CODE REVIEW

File: src/ralph/admin/sitetrees.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 )
 
 from ralph.admin.sites import ralph_site
+from ralph.data_importer.management.commands.importer import APP_MODELS
```
Comment (ar4s) [inline src/ralph/admin/sitetrees.py]: Move `APP_MODELS` to new module like `ralph.admin.helpers`

Comment (mkurek) [inline src/ralph/admin/sitetrees.py]: it's only one line so we can redefine it here. besides that - it's definition (`APP_MODELS = {model._meta.model_name: model for model in apps.get_models()}`) breaks when there will be model with the same name in more than one app

File: src/ralph/admin/helpers.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.apps import apps
+
+APP_MODELS = {model._meta.model_name: model for model in apps.get_models()}
```
Comment (mkurek) [inline src/ralph/admin/helpers.py:3]: it breaks when there will be model with the same name in more than one app

**Comment by ghost** [comment]
Change page titles come from model instead of sitetree.

**Pull request closed without merging**